packages/in-app-notification-manager/.env

```diff
@@ -0,0 +1,25 @@
+HOST=0.0.0.0
+PORT=4000
```

Comment:
4000 already in use by `m2m` gateway
```suggestion
PORT=4100
```

Reply:
I suggest 4200, I'm already using 4100 in PRs for the notification-config-process